metadata-ingestion/src/datahub/ingestion/source/ldap.py

```diff
@@ -370,6 +406,7 @@ def build_corp_user_mce(
     def build_corp_group_mce(self, attrs: dict) -> Optional[MetadataChangeEvent]:
         """Creates a MetadataChangeEvent for LDAP groups."""
         cn = attrs.get(self.config.group_attrs_map["urn"])
+        print(cn)
```

Comment:
revert this change

Reply:
I have made two  changes:
1) replaced  "self.report_stale_entity_soft_deleted(dn)" by  "self.dropped_dns.append(dn)"
2) removed "print(cn)"

I think my confusion was with the exact meaning of word "revert" in this context. Because there were so many changes at different times, so it was not clear to me what does "revert" mean? Should I revert to the original version of open source? Should I revert to my original version? Should I replace the code highlighted in "red" color by the code highlighted in "green" color as I usually do or should do exactly the opposite now because there is a word "revert" there?

I think just a simple replace this "..." by "..." because of "..." would be a way more easy to understand. Sorry, just some suggestions.